src/components/Crop.tsx

```diff
@@ -78,7 +78,7 @@ class Crop extends Component<Props> {
     if (!this.crop) return;
     // @ts-ignore
     interact(this.crop)
-      .draggable({})
+    .draggable({})
```

Comment:
Do run `yarn format` once.
You may want to set up your editor to auto-run prettier on save.